app/index.html

```diff
@@ -31,6 +31,18 @@
 </script>
 
 <script src="/__load/-/only=scripts/wikia.ext.instantGlobals,instantGlobalsOverride,abtesting,abtest"></script>
+<script>
+	/**
+	 * FIXME FEATURED VIDEO A/B TEST ONLY
+	 *
+	 * This is the earliest moment when we can get AB group
+	 * Normally we prerender video placeholder, but in this specific variant of AB Testing video is
+	 * placed differently so we have to hide the prerendered placeholder.
+	 */
+	if (window.Wikia.AbTest.inGroup('FEATURED_VIDEO_VIEWABILITY_VARIANTS', 'PAGE_PLACEMENT')) {
```

Comment:
We are able to set css per AB test variant in out AbTesting tool

Reply:
we agreed to not do that, since it's not fully released and scripts included in AB test framework might impact actual production